extensions/ql-vscode/test/vscode-tests/cli-integration/skeleton-query-wizard.test.ts

```diff
@@ -12,62 +12,55 @@ import { CancellationTokenSource } from "vscode-jsonrpc";
 import { testCredentialsWithStub } from "../../factories/authentication";
 import { DatabaseItem, DatabaseManager } from "../../../src/local-databases";
 import * as databaseFetcher from "../../../src/databaseFetcher";
-import { Credentials } from "../../../src/common/authentication";
-import { getErrorMessage } from "../../../src/pure/helpers-pure";
 
 describe("SkeletonQueryWizard", () => {
-  let mockCli: CodeQLCliServer;
-  let mockDatabaseManager: DatabaseManager;
   let wizard: SkeletonQueryWizard;
   let dir: tmp.DirResult;
   let storagePath: string;
-  let credentials: Credentials;
   let quickPickSpy: jest.SpiedFunction<typeof window.showQuickPick>;
   let generateSpy: jest.SpiedFunction<
     typeof QlPackGenerator.prototype.generate
   >;
   let createExampleQlFileSpy: jest.SpiedFunction<
     typeof QlPackGenerator.prototype.createExampleQlFile
   >;
-  let chosenLanguage: string;
   let downloadGitHubDatabaseSpy: jest.SpiedFunction<
     typeof databaseFetcher.downloadGitHubDatabase
   >;
   let openTextDocumentSpy: jest.SpiedFunction<
     typeof workspace.openTextDocument
   >;
 
+  const token = new CancellationTokenSource().token;
+  const credentials = testCredentialsWithStub();
+  const chosenLanguage = "ruby";
+  const mockDatabaseManager = mockedObject<DatabaseManager>({
+    setCurrentDatabaseItem: jest.fn(),
+    digForDatabaseItem: jest.fn(),
+  });
+  const mockCli = mockedObject<CodeQLCliServer>({
+    resolveLanguages: jest
+      .fn()
+      .mockResolvedValue([
```

Comment:
This mock will be reset when a new test is started, so I don't think you can move this out of the `beforeEach`. These calls are also not expensive, so I don't see why this would help with the timeout.

Reply:
My understanding was that because we were creating the object once, it would shorten the overall duration of the tests in the file. Since I went with increasing the timeout, I've moved the mock back in the forEach.

Reply:
I ended up doing this in the new refactoring PR, to avoid having to rebase that PR later: https://github.com/github/vscode-codeql/pull/2292/commits/d720bae20110630f0a51287041f8f58c4672be1e